M/MAGMA/build_tarballs.jl

```diff
@@ -1,38 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+include(joinpath(@__DIR__, "..", "..", "platforms", "cuda.jl"))
+
 name = "MAGMA"
-version = v"2.6.2"
+version = v"2.7.0"
+
+# Collection of sources required to complete build
 sources = [
-    GitSource("https://bitbucket.org/icl/magma", "864fa8b28e8c2905a41fbef32f156695378dc104"),
+    ArchiveSource("http://icl.utk.edu/projectsfiles/magma/downloads/magma-2.7.0.tar.gz", "fda1cbc4607e77cacd8feb1c0f633c5826ba200a018f647f1c5436975b39fd18"),
     DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/magma/
-
+cd $WORKSPACE/srcdir/magma-*
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/magma*

```
That way it'll work unchanged even if you need to switch back to a git snapshot in a future update.